### ISSUE TITLE
Covalence dust in Steve's cart wood cutter, stops it from planting saplings.

### ISSUE THREAD

**Issue by Tolana** [FTBTeam/FTB-Modpack-Issues#5350]
### Modpack

FTB NeoTech

### Modpack version

1.1.0

### Pack has been modified

- [ ] Yes

### Describe the bug

If you put covalence dust into the repair slot of the wood cutter, it stops planting saplings, and will just drive around. 

### Steps to reproduce

1. Make a simple steve's cart with an engine, and wood cutter.
2. Fill the cart with saplings, and fuel the cart if applicable 
3. Observe the cart planting saplings
4. Put the correct covalence dust into the repair slot
5. Observe the cart stop planting saplings

### Expected behaviour

Expected to work as normal when not in need of repair, and when in need repair and continue after.

**Comment by Saereth** [comment]
This is not a bug, its a feature of the mod lol we've requested this be a config option we can change in the future and the developers have agreed so once that is sorted on the mod side I'll include it and update it in the pack

**Comment by Zenith-One** [comment]
It's really confusing because following the questline tells you to put in covalence dust, which then makes the machine not do the thing it was built to do. This isn't explained anywhere. I recommend changing the quest to reflect this maybe?

**Comment by 630R63** [comment]
Just encountered this bug myself. I was googling up and down for what I was doing wrong because all the YouTube videos I watched were doing the same things at me, but for those people, the carts were just working. I couldn't understand why mine wasn't planting. It planted once when I put it down on the tracks and then never again. Until I realized that it was because of the covalence dust. 

I like how the issue was closed without actually updating the mod or modpack. The issue still happens on the 1.7.0 version.
